src/installing-workflow/configuring-object-storage.md

```diff
@@ -18,7 +18,7 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
     * Note: If you provide credentials with sufficient access to the underlying storage, Workflow components will create the buckets if they do not exist.
 * **Step 2:** If applicable, generate credentials that have write access to the storage buckets created in Step 1
 * **Step 3:** If you haven't already fetched the helm classic chart, do so with `helmc fetch deis/workflow-v2.0.0`
-* **Step 4:** Update storage details either by setting the appropriate environment variables or by modifying the template file `tpl/generate_params.toml`
+* **Step 4:** Update storage details either by setting the appropriate environment variables or by modifying the template file `tpl/generate_params.toml` available at `~/.helmc/workspace/charts/workflow-v2.0.0`
```

Comment:
Maybe reference `HELMC_HOME` here instead of `~/.helmc`? @krancour might also have a better suggestion here for file location references via env vars.